lib/OpenQA/WebAPI/Controller/Running.pm

```diff
@@ -231,7 +221,18 @@ sub streaming {
             }
         });
 
-    $self->on(finish => sub { Mojo::IOLoop->remove($id) });
+    # tell worker to create live stream
+    OpenQA::Utils::log_debug('Telling worker to start providing livestream');
+    $ipc->websockets('ws_send', $worker->id, 'livelog_start', $job->id);
```

Comment:
Please wrap it in a ```try/catch``` block or an ```eval{}``` to handle possible errors.

Even better, possibly this can be done in OpenQA::IPC, but it's out of the scope of this PR

Reply:
This uses `OpenQA::IPC->ipc`, what do you mean with `OpenQA::IPC`?

I don't see such error handling in most occurrences of such calls, but I could print a warning in the error case (not sure how to get an error message in the catch).

Reply:
I meant that try/catch should be in  ```OpenQA::IPC's _dispatch()```

Reply:
Ok, I can do that. However, I would create another PR because it has not a lot to do which this one. Note that the IPC call has not been introduced with this PR, but just moved.

Reply:
Like this https://github.com/os-autoinst/openQA/pull/1461?

Reply:
Yup :)

Reply:
Yeah i know it is just a 'move', but that made me notice some improvement that can be done here ( you are free to ignore them, as it is out of the scope of this PR, but you was so fast! 👍 👍  )